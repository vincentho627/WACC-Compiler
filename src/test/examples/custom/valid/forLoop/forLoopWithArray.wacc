# For loop which uses its iterator to index into an array

# Expected output:
# roses
# peonies
# hydrangea
# chrysanthemum
# lily
# sunflower
# daisy
# orchid
# tulip
# baby breath

begin
  string[] flowers = ["roses", "peonies", "hydrangea", "chrysanthemum", "lily",
  "sunflower", "daisy", "orchid", "tulip", "baby breath"];
   for i in 0 until 10 do
      println flowers[i]
    done
end
