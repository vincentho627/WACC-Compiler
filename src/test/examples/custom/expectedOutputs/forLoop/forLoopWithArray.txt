roses
peonies
hydrangea
chrysanthemum
lily
sunflower
daisy
orchid
tulip
baby breath
